js/client/modules/@arangodb/testutils/instance.js

```diff
@@ -509,24 +509,29 @@ class instance {
   // //////////////////////////////////////////////////////////////////////////////
 
   readAssertLogLines () {
-    const buf = fs.readBuffer(this.logFile);
-    let lineStart = 0;
-    let maxBuffer = buf.length;
+    try {
+      const buf = fs.readBuffer(this.logFile);
+      let lineStart = 0;
+      let maxBuffer = buf.length;
 
-    for (let j = 0; j < maxBuffer; j++) {
-      if (buf[j] === 10) { // \n
-        const line = buf.asciiSlice(lineStart, j);
-        lineStart = j + 1;
-        
-        // scan for asserts from the crash dumper
-        if (line.search('{crash}') !== -1) {
-          if (!IS_A_TTY) {
-            // else the server has already printed these:
-            print("ERROR: " + line);
+      for (let j = 0; j < maxBuffer; j++) {
+        if (buf[j] === 10) { // \n
+          const line = buf.asciiSlice(lineStart, j);
+          lineStart = j + 1;
+
+          // scan for asserts from the crash dumper
+          if (line.search('{crash}') !== -1) {
+            if (!IS_A_TTY) {
+              // else the server has already printed these:
+              print("ERROR: " + line);
+            }
+            this.assertLines.push(line);
           }
-          this.assertLines.push(line);
         }
       }
+    } catch (ex) {
+      print("failed to read " + this.logFile + " -> " + ex);
+      this.assertLines.push("failed to read " + this.logFile + " -> " + ex);
```

Comment:
`this.assertLines` seems to contain both success (ie, successfully reading a crash line from the log) and error messages (could not read line or file), without a clear key to help distinguish one kind from the other. Wondering if this could be problematic.